aptos-move/framework/aptos-framework/sources/configs/Stake.move

```diff
@@ -786,16 +864,31 @@ module AptosFramework::Stake {
         let active_validators = Vector::empty();
         validator_perf.num_blocks = 1;
         validator_perf.missed_votes = Vector::empty();
+
```

Comment:
curious why line 865 the num_blocks is 1 instead of 0?

Reply:
Good catch! It should be 0. on_new_epoch() is called after update_performance_statistics (which increments the number of blocks) in Block#prologue. So this should start out as 0 for a new epoch but immediately incremented to 1 once update_performance_statistics is called.

Reply:
Good catch! This should be 0. I was thinking 1 because the rewards distribution in on_new_epoch would fail with division by 0 - but this should not happen outside of tests as update_performance_statistics is always called after on_new_epoch in Block#prologue.

Reply:
Updated to be 0 now + updated tests correspondingly.

Reply:
now we know that we need to call update_performance_statistics before on_new_epoch 😄

Reply:
Yes it makes more sense now.